src/RudderClient.ts

```diff
@@ -193,19 +193,26 @@ async function group(groupId: string, traits: Object | null = null) {
     logWarn("group: 'groupId' must be a string");
     return;
   }
-  logWarn("group: Method not supported");
+  bridge.group(groupId, traits, options)
 }
 
-async function alias(previousId: string, userId: string) {
-  if (previousId == undefined) {
-    logWarn("alias: Mandatory field 'previousId' missing");
+async function alias(previousId: string, userId: string): Promise<void>; 
+async function alias(newId: string, options: Object | null = null) {
+  if (newId == undefined) {
+    logWarn("alias: Mandatory field 'newId' missing");
     return;
   }
-  if (typeof previousId != "string") {
-    logWarn("alias: 'previousID' must be a string");
+  if (typeof newId != "string") {
+    logWarn("alias: 'newId' must be a string");
     return;
   }
-  logWarn("alias: Method not supported");
+  if (typeof options == "string") {
+    bridge.alias(options, null)
+  } else if (typeof options == "object") {
+    bridge.alias(newId, options)
+  } else {
+    bridge.alias(newId, null)
+  }
```

Comment:
Can we combine the condition check for string and else into a single block and change it to the following way?

```
if (typeof options == "object") {
    bridge.alias(newId, options)
  } else {
    bridge.alias(newId, null)
  }
```